src/main/java/org/broadinstitute/hellbender/tools/walkers/validation/EvaluateInfoFieldConcordance.java

```diff
@@ -0,0 +1,140 @@
+    protected String truthInfoKey = GATKVCFConstants.CNN_2D_KEY;
+    @Argument(fullName="epsilon", shortName="epsilon", doc="Difference tolerance", optional=true)
+    protected double epsilon = 0.1;
+    private final EnumMap<ConcordanceState, MutableLong> snpCounts = new EnumMap<>(ConcordanceState.class);
+    private final EnumMap<ConcordanceState, MutableLong> indelCounts = new EnumMap<>(ConcordanceState.class);
+
+    private double snpSumDelta = 0.0;
+    private double snpSumDeltaSquared = 0.0;
+    private double indelSumDelta = 0.0;
+    private double indelSumDeltaSquared = 0.0;
+
+    public void onTraversalStart() {
+        for (ConcordanceState state : ConcordanceState.values()) {
+            this.snpCounts.put(state, new MutableLong(0));
+            this.indelCounts.put(state, new MutableLong(0));
+        }
+    }
+
+    protected void apply(AbstractConcordanceWalker.TruthVersusEval truthVersusEval, ReadsContext readsContext, ReferenceContext refContext) {
+        ConcordanceState concordanceState = truthVersusEval.getConcordance();
+        if (truthVersusEval.getTruthIfPresentElseEval().isSNP()) {
+            this.snpCounts.get(concordanceState).increment();
+        } else {
+            this.indelCounts.get(concordanceState).increment();
+        }
+        switch (concordanceState) {
+            case TRUE_POSITIVE: {
+                this.infoDifference(truthVersusEval.getEval(), truthVersusEval.getTruth());
+                break;
+            }
+            case FALSE_POSITIVE:
+            case FALSE_NEGATIVE:
+            case FILTERED_TRUE_NEGATIVE:
+            case FILTERED_FALSE_NEGATIVE: {
+                break;
+            }
+            default: {
+                throw new IllegalStateException("Unexpected ConcordanceState: " + concordanceState.toString());
+            }
+        }
+    }
+
+    private void infoDifference(VariantContext eval, VariantContext truth) {
+        double evalVal = Double.valueOf((String)eval.getAttribute(this.evalInfoKey));
+        double truthVal = Double.valueOf((String)truth.getAttribute(this.truthInfoKey));
+        double delta = evalVal - truthVal;
+        double deltaSquared = delta * delta;
+        if (eval.isSNP()) {
+            this.snpSumDelta += Math.sqrt(deltaSquared);
+            this.snpSumDeltaSquared += deltaSquared;
+        } else if (eval.isIndel()) {
+            this.indelSumDelta += Math.sqrt(deltaSquared);
+            this.indelSumDeltaSquared += deltaSquared;
+        }
+        if (Math.abs(delta) > this.epsilon) {
+            this.logger.warn(String.format("Difference (%f) greater than epsilon (%f) at %s:%d %s:", delta, this.epsilon, eval.getContig(), eval.getStart(), eval.getAlleles().toString()));
+            this.logger.warn(String.format("\t\tTruth info: " + truth.getAttributes().toString()));
+            this.logger.warn(String.format("\t\t Eval info: " + eval.getAttributes().toString()));
+        }
```

Comment:
Since the intended use here is just for debugging I'd prefer not write them to a file, we could make this a oneTime logger or just remove, although I do think the current behavior could be helpful in the future.